### DISCUSSION TITLE
Example for glTF

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3251] [Q&A] [answered]
Hello. 
Is there any working example on how to import and use glTF format files in the iTwin apps? I found the code of gltfDecorations. It would be helpful if there is also a sample app for this. 
https://www.itwinjs.org/learning/frontend/viewdecorations/#gltf-decorations

Here is the sample code.

**Comment by pmconne** [comment, marked as answer]
I expect @Ellord207 and @Josh-Schifter will add a sample or sandbox at some point - this is one of many brand new features in 3.0.
What exactly is missing from the example decorator though?

**Reply by hanan-commits-work** [reply]
Example decorator is complete as it is. I was just wondering if there is any complete sample app

**Reply by Ellord207** [reply]
> @Ellord207 and @Josh-Schifter will add a sample or sandbox at some point

An official sample for this is on the way.  I can create a sandbox if you need that can stand-in until the sample is finished.

**Reply by hanan-commits-work** [reply]
That would be helpful thank you.

**Reply by Ellord207** [reply]
Here is the sandbox: https://www.itwinjs.org/sandboxes/NathanElrod/glTF%20Decorator
